### ISSUE TITLE
what's the purpose of add_gt_proposals in RPNPostProcessor?

### ISSUE THREAD

**Issue by LostDawn** [facebookresearch/maskrcnn-benchmark#1231]
## ❓ Questions and Help
There is a function called add_gt_proposals in RPNPostProcessor, located at rpn/inference.py. It seems like trying to add target bounding boxes to proposed bounding boxes. But if so, the model will directly get the answer, and as a result, bounding boxes loss will be very low, nearly zero. Then the model can't get trained properly. Then why we call this function?

**Comment by yww211** [comment]
https://github.com/pytorch/vision/blob/ab73b44887eb9d7566b2a80fd45cb5f68b11d643/torchvision/models/detection/roi_heads.py#L642 
This function also exsits in roiheads.py.
